### PR TITLE
added missing description text on 3rd layer, changed description of s…

### PR DESCRIPTION
**Pull request by expeditionengineer**
…earch results

### CODE REVIEW

File: webcentral/src/webcentral_app/static/js/criteriaCatalogFunctions.js
Comment (c0nb4) [inline webcentral/src/webcentral_app/static/js/criteriaCatalogFunctions.js:1]: Why is the catch statement empty?

File: webcentral/src/StartSearch/views.py
Comment (c0nb4) [inline webcentral/src/StartSearch/views.py:1]: LGTM

**Comment by c0nb4** [comment]
@expeditionengineer selenium tests are failing

**Comment by expeditionengineer** [comment]
test of criteria catlog in search result test should be fixed now

File: webcentral/src/webcentral_app/static/js/criteriaCatalogFunctions.js
Comment (c0nb4) [reply webcentral/src/webcentral_app/static/js/criteriaCatalogFunctions.js:1]: It it possible to change to this? 

`catch (error) {
    // Handle the error here, e.g., log it or display a message
    console.error("An error occurred:", error);
}`

Comment (expeditionengineer) [reply webcentral/src/webcentral_app/static/js/criteriaCatalogFunctions.js:1]: I added the code block

**Review by c0nb4 — APPROVED** [review]

**Pull request merged**